### PR TITLE
 DO NOT MERGE: Secureboot signing support rc2 - ws signed

### PR DESCRIPTION
**Pull request by bofferdn**
This change is [Reviewable](https://reviewable.io/reviews/open-power/op-build/1308)

### CODE REVIEW

**Comment by stewart-ibm** [comment]
Can one of the admins verify this patch for testing?

**Comment by bofferdn** [comment]
This has actually passed all tests but some runs ran into infrastructure issues.  Would like to get a review of all the non-48 MB patch content before the final drop (when the pnor pointer will have to change, etc.)

File: openpower/package/openpower-pnor/openpower-pnor.mk [hunk 1]
```diff
@@ -4,8 +4,10 @@
 #
 ################################################################################
 
-OPENPOWER_PNOR_VERSION ?= 7dd93556bcefdb96483e413bbd650e3929db2769
-OPENPOWER_PNOR_SITE ?= $(call github,open-power,pnor,$(OPENPOWER_PNOR_VERSION))
+#OPENPOWER_PNOR_VERSION ?= e9e092f5ebf6baa7170b9bb6c5ec681a9f6174e3
+#OPENPOWER_PNOR_SITE ?= $(call github,open-power,pnor,$(OPENPOWER_PNOR_VERSION))
+OPENPOWER_PNOR_VERSION ?= 4f81780f6491b2a9eb76bd812be3faf9b42ecb71
```
Comment (bofferdn) [inline openpower/package/openpower-pnor/openpower-pnor.mk]: after logical approval, and pnor project updated, will fix the pointer

**Comment by bofferdn** [comment]
superseded by PR #1323

**Pull request closed without merging**